src/Table.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+export const Table = styled.table`
+  border-collapse: collapse;
+
+  th {
+    text-align: left;
```

Comment:
This one is by default, so can be ommited

Reply:
No

Reply:
Out of curiosity - why? :)

Reply:
Because you're wrong and for `th` default align — center

Reply:
https://output.jsbin.com/wuluqajexu

Reply:
Indeed!
Good insight. Sorry for confusion, I really believed it was left aligned...